docs/docs/configuration/index.md

```diff
@@ -502,7 +502,7 @@ ui:
   # Optional: Set the default live mode for cameras in the UI (default: shown below)
   live_mode: mse
   # Optional: Set a timezone to use in the UI (default: use browser local time)
-  timezone: None
+  # timezone: None
```

Comment:
I would remove the None as well unless it's a valid value. You could replace it with a valid timezone string as an example.